assets/app/view/game/autoroute.rb

```diff
@@ -0,0 +1,14 @@
+module View
+  module Game
+    class Autoroute
+      def self.calculate(game, corporationId)
```

Comment:
always use snake_case

Reply:
Heh! Of course. I haven't run rubocop on this but obviously I will before submitting this for real.